trezor/trezor-suite
# Fix PostCSS line return parsing error

## Description

- update storybook to get rid of `webpack4` storybook dependency which had old version of `postcss`
- remove `@storybook/core` as it was not used after migration to v7
- replace `stylelint-processor-styled-components` by `postcss-styled-syntax` which is not deprecated and is recommended by stylelint docs and also does not include old `postcss`
- update `stylelint` to latest version and update it's config
- fix all css problems in Suite package
- disable stylelint in components package https://github.com/trezor/trezor-suite/issues/9810
- use one `.stylelintrc` config from root
- add stylelint check to `suite-web`, `suite-desktop-ui`, `connect-explorer`, `connect-popup`, `connect-popup-ui`

## Related Issue

https://github.com/trezor/trezor-suite/security/dependabot/189
closes https://github.com/trezor/trezor-suite/issues/7613
https://github.com/trezor/trezor-suite/pull/8872
